src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -492,7 +492,7 @@
   </data>
   <data name="RestoreCommandPackageRestoreOptOutMessage" xml:space="preserve">
     <value>Restoring NuGet packages...
-To prevent NuGet from downloading packages during build, open the Visual Studio Options dialog, click on the Package Manager node and uncheck '{0}'.</value>
+To prevent NuGet from downloading packages during build, open the Visual Studio Options dialog, click on the NuGet Package Manager node and uncheck '{0}'.</value>
```

Comment:
There might be some tests depending on this string, double check that or wait for all tests to pass.